README.md

```diff
@@ -1,2 +1,8 @@
 # Services-Courses
+
 .NET 7 Minimal API for Microservices
+
+## Will Miller Microservices
+
+> 1. Some Sentence 1
+> 1. Some Sentence 2
```

Comment:
Could you explain this sentence further?

Reply:
Sure.